diff --git a/node_modules/tocbot/src/js/build-html.js b/node_modules/tocbot/src/js/build-html.js
index 40ed68a..cf52283 100644
--- a/node_modules/tocbot/src/js/build-html.js
+++ b/node_modules/tocbot/src/js/build-html.js
@@ -92,7 +92,8 @@ module.exports = function (options) {
       SPACE_CHAR + 'node-name--' + data.nodeName +
       SPACE_CHAR + options.extraLinkClasses)
     item.appendChild(a)
-    return item
+    // return item
+    return a
   }
 
   /**
@@ -101,7 +102,8 @@ module.exports = function (options) {
    * @return {HTMLElement}
    */
   function createList (isCollapsed) {
-    var listElement = (options.orderedList) ? 'ol' : 'ul'
+    // var listElement = (options.orderedList) ? 'ol' : 'ul'
+    var listElement = 'div'
     var list = document.createElement(listElement)
     var classes = options.listClass +
       SPACE_CHAR + options.extraListClasses
